sql/gameserver_dev_Full.sql

```diff
@@ -769,79 +769,79 @@ CREATE TABLE IF NOT EXISTS `npc` (
-  `skill5` int(11) NOT NULL DEFAULT '0',
-  `skill6` int(11) NOT NULL DEFAULT '0',
-  `skill7` int(11) NOT NULL DEFAULT '0',
-  `skill8` int(11) NOT NULL DEFAULT '0',
-  `skill9` int(11) NOT NULL DEFAULT '0',
-  `skill10` int(11) NOT NULL DEFAULT '0',
-  `skill11` int(11) NOT NULL DEFAULT '0',
-  `skill12` int(11) NOT NULL DEFAULT '0',
-  `skill13` int(11) NOT NULL DEFAULT '0',
-  `skill14` int(11) NOT NULL DEFAULT '0',
-  `skill15` int(11) NOT NULL DEFAULT '0',
-  `skill16` int(11) NOT NULL DEFAULT '0',
-  `skill17` int(11) NOT NULL DEFAULT '0',
-  `skill18` int(11) NOT NULL DEFAULT '0',
-  `skill19` int(11) NOT NULL DEFAULT '0',
-  `skill20` int(11) NOT NULL DEFAULT '0',
-  `skill21` int(11) NOT NULL DEFAULT '0',
-  `skill22` int(11) NOT NULL DEFAULT '0',
-  `skill23` int(11) NOT NULL DEFAULT '0',
-  `skill24` int(11) NOT NULL DEFAULT '0',
-  `skill25` int(11) NOT NULL DEFAULT '0',
-  `skill26` int(11) NOT NULL DEFAULT '0',
-  `skill27` int(11) NOT NULL DEFAULT '0',
-  `skill28` int(11) NOT NULL DEFAULT '0',
-  `skill29` int(11) NOT NULL DEFAULT '0',
-  `skill30` int(11) NOT NULL DEFAULT '0',
-  `skill31` int(11) NOT NULL DEFAULT '0',
-  `skill32` int(11) NOT NULL DEFAULT '0',
-  `skill33` int(11) NOT NULL DEFAULT '0',
-  `skill34` int(11) NOT NULL DEFAULT '0',
-  `skill35` int(11) NOT NULL DEFAULT '0',
-  `skill36` int(11) NOT NULL DEFAULT '0',
-  `skill37` int(11) NOT NULL DEFAULT '0',
-  `skill38` int(11) NOT NULL DEFAULT '0',
-  `skill39` int(11) NOT NULL DEFAULT '0',
-  `skill40` int(11) NOT NULL DEFAULT '0',
-  `skill41` int(11) NOT NULL DEFAULT '0',
-  `skill42` int(11) NOT NULL DEFAULT '0',
-  `skill43` int(11) NOT NULL DEFAULT '0',
-  `skill44` int(11) NOT NULL DEFAULT '0',
-  `skill45` int(11) NOT NULL DEFAULT '0',
-  `skill46` int(11) NOT NULL DEFAULT '0',
-  `skill47` int(11) NOT NULL DEFAULT '0',
-  `skill48` int(11) NOT NULL DEFAULT '0',
-  `skill49` int(11) NOT NULL DEFAULT '0',
-  `skill50` int(11) NOT NULL DEFAULT '0',
-  `skill51` int(11) NOT NULL DEFAULT '0',
-  `skill52` int(11) NOT NULL DEFAULT '0',
-  `skill53` int(11) NOT NULL DEFAULT '0',
-  `skill54` int(11) NOT NULL DEFAULT '0',
-  `skill55` int(11) NOT NULL DEFAULT '0',
-  `skill56` int(11) NOT NULL DEFAULT '0',
-  `skill57` int(11) NOT NULL DEFAULT '0',
-  `skill58` int(11) NOT NULL DEFAULT '0',
-  `skill59` int(11) NOT NULL DEFAULT '0',
-  `skill60` int(11) NOT NULL DEFAULT '0',
-  `skill61` int(11) NOT NULL DEFAULT '0',
-  `skill62` int(11) NOT NULL DEFAULT '0',
-  `skill63` int(11) NOT NULL DEFAULT '0',
-  `skill64` int(11) NOT NULL DEFAULT '0',
-  `skill65` int(11) NOT NULL DEFAULT '0',
-  `skill66` int(11) NOT NULL DEFAULT '0',
-  `skill67` int(11) NOT NULL DEFAULT '0',
-  `skill68` int(11) NOT NULL DEFAULT '0',
-  `skill69` int(11) NOT NULL DEFAULT '0',
-  `skill70` int(11) NOT NULL DEFAULT '0',
-  `skill71` int(11) NOT NULL DEFAULT '0',
-  `skill72` int(11) NOT NULL DEFAULT '0',
+  `skill0` tinyint(4) unsigned NOT NULL DEFAULT '0',
```

Comment:
no need for 4 bytes, it can be placed all in 1 byte, it's only 0 and 1, (have skill or not)